packages/suite/src/actions/suite/routerActions.ts

```diff
@@ -91,8 +94,8 @@ export const onAnchorChange = (anchor?: AnchorType) => (dispatch: Dispatch, _get
 export const init = () => (dispatch: Dispatch, getState: GetState, extra: ExtraDependencies) => {
     // check if location was not already changed by initialRedirection
     if (getState().router.app === 'unknown') {
-        const location = extra.routerServices.getLocation();
-        const url = location.pathname + location.hash;
+        const { pathname, search, hash } = extra.routerServices.getLocation();
+        const url = `${pathname}${search ?? ''}${hash ?? ''}`;
```

Comment:
I guess `getLocation` is in fact useless, it is the state of th router, can be pulled from here.